notebooks/matrix_tube_pipeline_seqcount_norm.ipynb

```diff
@@ -388,6 +391,8 @@
     "# initialize a new PlateReplication object to manage metadata, conversions, and more for you.\n",
     "# initialize w/preferred well_col.\n",
     "well_col = 'Library Well'\n",
+    "# 'Library Well' differs from 'Well' because the later specifies the gDNA source well while the former\n",
+    "# specifies the well (destination well) that will contain the sequencing library for the sample.\n",
```

Comment:
Question: does this mean `Well` is a 96-well plate position and `Library Well` is a 384-well plate position?

Reply:
not exactly. The 'Well' column in the plate_df tracks the well location, in 384-well format, where the gDNA of a sample is contained while the 'Library Well' tracks the well location for the sequencing library of a sample. Most of the time the 'Well' and the 'Library Well' will match, but that is not always the case. For instance when doing plate replication, a sample from 1 'Well' can be replicated and placed in up to 4 different 'Library Wells'.

Reply:
thanks!